### DISCUSSION TITLE
Regarding Uploadation

### DISCUSSION THREAD

**Discussion by Harshu467** [makarand-20/EPAM_Home_Tasks#48] [Q&A] [answered]
I have a question regarding uploadation of files in task number 2 . Which files are supposed to be uploaded in task number 2 ? Please Answer it and thank you for kind answer to whoever.....?????

**Comment by makarand-20** [comment, marked as answer]
You should upload the separate .js files there...

**Reply by Harshu467** [reply]
Thank you...
